example.js

```diff
@@ -1,4 +1,4 @@
-const libui = require('.');
+const libui = require('./index.js');
```

Comment:
Why this change? `require('.')` should be independent by the cwd and equivalent to `require('./index.js')`